test/set.js

```diff
@@ -11,23 +11,93 @@ describe('Set functions', function() {
   describe('#.intersection', function() {
 
     it('should properly compute the intersection of two sets.', function() {
-      var A = new Set([1, 2, 3]),
-          B = new Set([2, 3, 4]);
+      var Exemplar = new Set([2, 3, 1]);
```

Comment:
Those variables should not be capitalized.

Reply:
I updated all, including the pre-existing ones, to all be lowerCameled and more than one character